core/src/core/block.rs

```diff
@@ -63,6 +63,10 @@ pub enum Error {
 	InvalidPow,
 	/// Kernel not valid due to lock_height exceeding block header height
 	KernelLockHeight(u64),
+	/// NRD kernels are not valid prior to HF3.
+	NRDKernelPreHF3,
+	/// NRD kernels are not valid if disabled locally via "feature flag".
+	NRDKernelNotEnabled,
```

Comment:
why not flip value and call it NRDKernelEnabled ?
or if you really want to keep same value, use NRDKernelDisabled (I prefer above though).